### PR TITLE
CVE-2015-5209

### PR DESCRIPTION
**Pull request by dwelcheck**
Republished pull request so it runs against CI

### CODE REVIEW

File: cves/CVE-2015-5209.yml [hunk 1]
```diff
@@ -1,38 +1,41 @@
...
-reported:
+reported: 
 
 announced_instructions: |
   Was there a date that this vulnerability was announced to the world? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE data.
   Please enter your date in YYYY-MM-DD format.
-announced:
+announced: 2015-09-06
+# https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61317918
```
Comment (lukasyelle) [inline cves/CVE-2015-5209.yml:38]: I like that you included comments referencing the source of your data! gg.

Comment (lukasyelle) [reply cves/CVE-2015-5209.yml:38]: add 5 upvotes 👍🏻

**Comment by dolanbernard** [comment]
While this is definitely an example of CWE-20, do you think that the more specific CWE-470 is a more precise explanation of this vulnerability? It looks like this vulnerability existed because users were able to access the 'top' object and use it to manipulate the root of the execution context. 
Additionally, I noticed there was a minor typo in your last paragraph. The answer to the 'mistakes' section contains the word 'critical' spelled as 'criitcal.'

**Comment by dolanbernard** [comment]
+2 upvotes for CVE-2015-5209

**Comment by dwelcheck** [comment]
> While this is definitely an example of CWE-20, do you think that the more specific CWE-470 is a more precise explanation of this vulnerability? It looks like this vulnerability existed because users were able to access the 'top' object and use it to manipulate the root of the execution context.
> Additionally, I noticed there was a minor typo in your last paragraph. The answer to the 'mistakes' section contains the word 'critical' spelled as 'criitcal.'

Nice find, wasn't aware of that CWE.  That does fit better, I'll change that and the typo.

**Comment by ghost** [comment]
Hello, I'm commenting on your CVE since a person in my cohort didn't finish theirs.
Anyway, I think your CVE is very well written but in the last answer I think you could explain some steps the developers should go through to prevent a bug like this from re-appearing in the future.

**Comment by dwelcheck** [comment]
> Hello, I'm commenting on your CVE since a person in my cohort didn't finish theirs.
> Anyway, I think your CVE is very well written but in the last answer I think you could explain some steps the developers should go through to prevent a bug like this from re-appearing in the future.

Ok, will do!

**Pull request merged**